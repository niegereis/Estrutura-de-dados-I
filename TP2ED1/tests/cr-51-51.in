51 51
r
***************************************************
*     *           *   *  M*         *       *   * *
* * *** ***** *** * * * *** ******* * *** * * * * *
* * *   *   *   *   * * *   *       *   * * * *   *
* *** ***** *** ******* * * ********* *** * * *** *
*   *         *       * * * *         *   *   * * *
* * ********* ******* * *** * ********* ******* * *
* * *       *   *   * *   * *   *     *   *     * *
* * * * * ***** *** * *** * *** * *** *** * *** * *
* *   * * *     *   *   * *     * * *     *   *   *
******* *** ***** ***** * * ***** * ********* *****
*     * *   *   *     * * * *     *     *     *   *
* *** * * *** * * *** * * * * ********* * ***** ***
*   *   * *   * *   * *   * * *       * *     *   *
* * *** * * *** * *** ******* ***** * * ***** *** *
* * *   *   * * * *           *     * *     *   * *
*** * ******* * *** *********** ***** ***** *** * *
*   * *       *     *   *       *   *         * * *
* *** *** *** ******* * * ******* *********** * * *
*   *     *     *     * *   *         * *     *   *
* * *** ********* * *** *** * ***** * * * ******* *
* * *   *     *   * * * *   * *     * * *   *     *
*** * *** *** * *** * * * *** * ***** * *** * *****
*   *   * * *   * * * *   * * *     *   *   * *   *
* ***** * * ***** * * ***** * ***** *** * *** * * *
* * *   * *   *   * * *       *   * *   * *   * * *
* * * *** * *** * * * * ******* * * ***** * *** * *
* *   *   *     * * * *   *     * * *   * * *   * *
* ***** ********* * * *** ******* * * * * * *** ***
*     *   *     * * *             *   *   *   *   *
* *** *** * *** * * *********** ********* *** * * *
*   *   * * * * * *       *     *       * *   * * *
*** * *** * * * * ******* ******* ***** *** ***** *
*   * *   * *   *       *   *     *   *   *       *
* *** * *** * ******* * *** * ***** ***** ******* *
* * *   *   *   *     * *   * *     *         *   *
* * ******* *** * ***** * * * *** * * ******* * ***
* *         *   * *   * * * *   * * * *     * * * *
* *** ******* *** * * *** ***** *** * * ***** * * *
*     *     * *   * *     *   *     * * *     *   *
******* * *** * * * ******* * *** *** * * ******* *
*       *   *   * *   *     *     *   * *       * *
* ********* ***** *** * *********** *** *** ***** *
*     *   *   *   * *   *         * *   *   *   * *
* ***** * * * * *** ************* * *** * *** * * *
* *     *   * *   *               *   *   *   * * *
* * ************* * ***** ******* *** ***** *** * *
* *     *   *   * * *     *       * *     * *   * *
* ***** * * * * * *** ***** ******* ***** * * *** *
*     *   *   *       *                 *   *      
***************************************************
