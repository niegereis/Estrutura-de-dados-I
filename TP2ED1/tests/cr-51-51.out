267
***************************************************
*     *ooooooooooo*ooo*ooM*         *    ooo*ooo* *
* * ***o*****o***o*o*o*o*** ******* * ***o*o*o*o* *
* * *ooo*   *ooo*ooo*o*o*   *       *   *o*o*o*ooo*
* ***o***** ***o*******o* * ********* ***o*o*o***o*
*   *ooooooooo*ooooooo*o* * *         *ooo*ooo* *o*
* * *********o*******o*o*** * *********o******* *o*
* * *       *ooo*   *o*ooo* *   *ooooo*ooo*ooooo*o*
* * * * * *****o*** *o***o* *** *o***o***o*o***o*o*
* *   * * *ooooo*   *ooo*o*     *o* *ooooo*ooo*ooo*
******* ***o***** *****o*o* *****o* *********o*****
*     * *ooo*ooo*     *o*o* *ooooo*     *ooooo*   *
* *** * *o***o*o* *** *o*o* *o********* *o***** ***
*   *   *o*ooo*o*   * *ooo* *o*       * *ooooo*   *
* * *** *o*o***o* *** *******o***** * * *****o*** *
* * *   *ooo* *o* *ooooooooooo*     * *     *ooo* *
*** * ******* *o***o*********** ***** ***** ***o* *
*   * *       *ooooo*   *       *   *         *o* *
* *** *** *** ******* * * ******* *********** *o* *
*   *     *     *     * *   *         * *     *ooo*
* * *** ********* * *** *** * ***** * * * *******o*
* * *   *     *   * * * *   * *     * * *   *ooooo*
*** * *** *** * *** * * * *** * ***** * *** *o*****
*   *   * * *   * * * *   * * *     *   *   *o*   *
* ***** * * ***** * * ***** * ***** *** * ***o* * *
* * *   * *   *   * * *       *   * *   * *ooo* * *
* * * *** * *** * * * * ******* * * ***** *o*** * *
* *   *   *     * * * *   *     * * *   * *o*   * *
* ***** ********* * * *** ******* * * * * *o*** ***
*     *   *     * * *             *   *   *ooo*   *
* *** *** * *** * * *********** ********* ***o* * *
*   *   * * * * * *       *     *       * *ooo* * *
*** * *** * * * * ******* ******* ***** ***o***** *
*   * *   * *   *       *   *     *   *   *ooooooo*
* *** * *** * ******* * *** * ***** ***** *******o*
* * *   *   *   *     * *   * *     *         *ooo*
* * ******* *** * ***** * * * *** * * ******* *o***
* *         *   * *   * * * *   * * * *     * *o* *
* *** ******* *** * * *** ***** *** * * ***** *o* *
*     *     * *   * *     *   *     * * *     *ooo*
******* * *** * * * ******* * *** *** * * *******o*
*       *   *   * *   *     *     *   * *       *o*
* ********* ***** *** * *********** *** *** *****o*
*     *   *   *   * *   *         * *   *   *   *o*
* ***** * * * * *** ************* * *** * *** * *o*
* *     *   * *   *               *   *   *   * *o*
* * ************* * ***** ******* *** ***** *** *o*
* *     *   *   * * *     *       * *     * *   *o*
* ***** * * * * * *** ***** ******* ***** * * ***o*
*     *   *   *       *                 *   *    oo
***************************************************
